docs/deployment/domain-configuration.md

```diff
@@ -4,11 +4,13 @@
 
 ```shell
 domains:add <app> DOMAIN                                                                     Add a domain to app
+domains:add-global <domain>                                                                  Add global domain name
 domains [<app>]                                                                              List domains
 domains:clear <app>                                                                          Clear all domains for app
 domains:disable <app>                                                                        Disable VHOST support
 domains:enable <app>                                                                         Enable VHOST support
 domains:remove <app> DOMAIN                                                                  Remove a domain from app
+domains:remove-global <domain>                                                               Remove global domain name
 domains:set-global <domain>                                                                  Set global domain name
```

Comment:
I think we'll want to drop `set-global` in favor of `add-global`/`remove-global`